yarn.lock

```diff
@@ -32,18 +32,32 @@ acorn-globals@^4.0.0:
   dependencies:
     acorn "^5.0.0"
 
-acorn@^5.0.0, acorn@^5.1.2:
```

Comment:
any idea where these `yarn.lock` changes are coming from? Maybe a different yarn version (I am on 1.3.2). Since there are no changes in package json, I would expect none here. You can fix it by using`git checkout master yarn.lock`.

Reply:
Hmm master yarn was not up to date either, just pushed.

Reply:
Silly me. I add eslint and prettier to almost every project. I removed it after. That should explain that I will fix it